x/validator/client/cli/query_disabled_validator.go

```diff
@@ -0,0 +1,88 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/utils/cli"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/x/validator/types"
+)
+
+func CmdListDisabledValidator() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "all-disabled-validators",
+		Short: "Query the list of all disabled validators",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx := client.GetClientContextFromCmd(cmd)
+
+			pageReq, err := client.ReadPageRequest(cmd.Flags())
+			if err != nil {
+				return err
+			}
+
+			queryClient := types.NewQueryClient(clientCtx)
+
+			params := &types.QueryAllDisabledValidatorRequest{
+				Pagination: pageReq,
+			}
+
+			res, err := queryClient.DisabledValidatorAll(context.Background(), params)
+			if cli.IsKeyNotFoundRPCError(err) {
+				return clientCtx.PrintString(cli.LightClientProxyForListQueries)
+			}
+			if err != nil {
+				return err
+			}
+
+			return clientCtx.PrintProto(res)
+		},
+	}
+
+	flags.AddPaginationFlagsToCmd(cmd, cmd.Use)
+	flags.AddQueryFlagsToCmd(cmd)
+
+	return cmd
+}
+
+func CmdShowDisabledValidator() *cobra.Command {
+	var address string
+
+	cmd := &cobra.Command{
+		Use:   "disabled-validator --address [address]",
```

Comment:
We have incosistent API names:
`tx validator add-node`, `query validator node`, but
`tx validator add-validator`, `query validator disabled-validator`, etc.

Need to make it consistent:
`tx validator disable-node`
`tx validator enable-node`
`tx validator propose-disable-node`
`tx validator approve-disable-node`
`query validator disabled-node`
`query validator proposed-disable-node`
`query validator all-disabled-node`
`query validator all-proposed-disable-node`